crates/full-node/sov-metrics/src/lib.rs

```diff
@@ -71,6 +71,11 @@ impl MaybeTimer {
     pub fn elapsed(&self) -> std::time::Duration {
         std::time::Duration::from_secs(0)
     }
+
+    /// Returns the elapsed time since the timer if the `native` feature is enabled. Otherwise does nothing.
+    pub fn stop_and_get_elapsed(&mut self) -> std::time::Duration {
```

Comment:
Looks like this method is only invoked from native code, so the stub is unnecessary.

Reply:
It fails to compile without it - possibly due to test dependencies